hyundai_kia_connect_api/ApiImpl.py

```diff
@@ -34,8 +36,8 @@ def login(self, username: str, password: str) -> Token:
         """Login into cloud endpoints and return Token"""
         pass
 
-    def get_vehicles(self, token: Token) -> list[Vehicle]:
-        """Return all Vehicle instances for a given Token"""
+    def get_vehicles(self, token: Token, vehicles: list[Vehicle]) -> None:
```

Comment:
I still think this is a bad idea, `get_vehicles` should get the vehicles, let's keep API functions as simple and lean as much as possible.